src/chat.ts

```diff
@@ -108,7 +108,7 @@ export class Chat {
     }
 
     // Move onto "persistent" messages
-    await this._summarize("persistent");
+    //await this._summarize("persistent");
```

Comment:
Why comment-out and not just remove? If you'd like to remove this, please detail why that's so in the PR so others know.